lib/kdump_utils.pm

```diff
@@ -172,8 +172,12 @@ sub activate_kdump {
             type_string $memory_kdump;
             wait_screen_change(sub { send_key 'ret' }, 10) for (1 .. 2);
             record_soft_failure 'default kdump memory size is too small for ppc64le and aarch64, see bsc#1161421';
+            $expect_restart_info = 1;
+        }
+        else {
+            wait_screen_change(sub { send_key 'ret' }, 10);
+            $expect_restart_info = 1 unless check_var('QEMURAM', '4096');
```

Comment:
this can go after the first if (outside the if..else) apparently.

Reply:
I thought the same and my initial code was like that, but apparently one of the 'ret' is used sometimes and sometimes not. Anyway the purpose of this PR is to remove the soft-failure if can be removed (Petr is checking). The code has two clear issues, (1) it utilizes the same logic to handle two different test condition actually using/processing 1 or 2 'ret's depending on the case (2) the workaround doesn't behave like a plugin, when you remove it, stop working.